### DISCUSSION TITLE
hide menu on birdseye view

### DISCUSSION THREAD

**Discussion by shawwellpete** [blakeblackshear/frigate#2988] [Ideas]
I am loving Frigate which, after a bit of work, I now have working really well with 4 [reolink camerras](https://blog.peterblandford.com/2021/10/30/reolink-4k-cameras-with-frigate-home-assistant-and-24-hour-recording/)

I have a question / feature request:

The birdseye wiew seems to be the best way for me to get a decent stream on a wall mounted ipad and works better than custom Frigate Card with WebRTC

I have noticed that below a certain browser size the frigate menu on the left is hidden. Is there any way of hiding the menu for any browser size?

I use birdseye within an i-frame in home assistant and would love the streams to be bigger...
![2022-03-19_11-10-08](https://user-images.githubusercontent.com/74064242/159118703-922c94a7-0f3b-4f16-bd44-69be8f949b26.jpg)

**Comment by NickM-27** [comment]
Wanted to note though that the frigate card can play the birdseye using the jsmpeg stream type, just fyi in case you haven't tried that

**Reply by shawwellpete** [reply]
Thanks Nick, I'll look into that

**Comment by rtw819** [comment]
Thinking outside of HA, and just for normal Browser use of the Birdseye page, would it be possible to allow the left menu bar to be minimized so that we can recover and use more screen real-estate for the camera streams instead of losing the entire left side to the menu?
